conda/base/context.py

```diff
@@ -1254,6 +1254,22 @@ def auto_activate_base(self) -> bool:
     def default_activation_env(self) -> str:
         return self._default_activation_env or ROOT_ENV_NAME
 
+    @property
+    def default_activation_prefix(self) -> PathType:
+        """Return the prefix of the default_activation_env.
+
+        If the default_activation_env is an environment name, get the corresponding
+        prefix; otherwise it is already a prefix, so just return it.
+
+        :return: Prefix of the default_activation_env
+        """
+        from ..exceptions import EnvironmentNameNotFound
+
+        try:
+            return locate_prefix_by_name(self.default_activation_env)
+        except EnvironmentNameNotFound:
+            return self.default_activation_env
```

Comment:
Checking if the current value is already an absolute path is a cheaper operation than checking all of the `envs_dirs` for a possible env so we should reorder this.